lib/hammer_cli_foreman/command_extensions/update_common.rb

```diff
@@ -7,7 +7,9 @@ class UpdateCommon < HammerCLI::CommandExtensions
 
       request_params do |params, command_object|
         update_params = params[command_object.resource.singular_name]
-        command_object.context[:action_message] = :nothing_to_do if update_params && update_params.empty?
+        update_params = params if update_params.nil?
+        update_params = update_params.values.select { |val| !val.nil?  }
```

Comment:
We can't do that, since it's correct to have e.g. `{ "id" => "1", "dns_id" => nil }` to reset/remove the dns relation of a resource. With this approach we will show `Nothing to update` even if we actually going to change/update something.